build.rs

```diff
@@ -27,26 +27,11 @@ fn generate_contract_abis() {
         ("AaveV2StablecoinCellar", "aave_v2_stablecoin"),
         ("CellarV1", "cellar_v1"),
         ("CellarV2", "cellar_v2"),
-        ("CellarV2_2", "cellar_v2_2"),
```

Comment:
Why are we deleting all of these that was just added earlier? Also why delete `abi/CellarV2_2.json`?

Reply:
This PR doesn't have handlers for new cellars yet, I think I originally added this by mistake